### ISSUE TITLE
Windows Malware Alert

### ISSUE THREAD

**Issue by w9ran** [berteh/ScribusGenerator#186]
After installing the stable version of Scribus (1.4.5) and downloading Scribus Generator for v1.4.x to 1.5.5 I get a malware warning when I unzip it:   Trojan:Script/Wacatac.B!ml

Is this a false alarm?  
Thanks, Bob

**Comment by berteh** [comment]
Hello Bob,

Provided you downloaded your copy from this GitHub repository then yes, it
is a false alarm:

https://github.com/berteh/ScribusGenerator/archive/master.zip for Scribus
v1.4.x to 1.5.5

https://github.com/berteh/ScribusGenerator/archive/python3.zip for Scribus
1.5.6+

I will look into this further, but you can safely run the script in the
meantime.

Thanks for this report, even though it's a bad news. First time I hear of
it about a zip file of uncompiled material.

**Comment by w9ran** [comment]
Thanks much for your prompt reply.  I since installed and have had success with the Scribus 1.5.7 development version, and the corresponding script downloaded from the same Github page without triggering the alert I got with the master.zip file.

I did fill the .log file last night until finally figuring out why Scribus Generator was not happy.   It seems that saving a file in OpenOffice Calc as .csv actually inserts semi-colons instead of commas as delimiters!    I've tried it on other data to confirm and if you examine their default .csv files in a text editor  you see nothing but semi-colons.  Here's a snippet:

> Robert;Nickels;1;PRM;EN52DG;42.2624202;-89.691601;Stephenson ;IL;

Thanks for providing the ability to change the delimiter in Scribus Generator as I haven't found a way to change it in OpenOffice Calc.   Apparently the definition of "C" in "csv" is not well understood by Apache!   

Bob
